src/app/screen-snippet-handler.ts

```diff
@@ -87,19 +89,55 @@ class ScreenSnippet {
    *
    * @param webContents {WeContents}
    */
-  public async capture(
-    webContents: WebContents,
-    currentWindow?: string,
-    hideOnCapture?: boolean,
-  ) {
+  public async capture(webContents: WebContents, hideOnCapture?: boolean) {
+    const currentWindowObj = BrowserWindow.getFocusedWindow();
+    const windowObj = winStore.getWindowStore();
+    const currentWindowName = (currentWindowObj as ICustomBrowserWindow)
+      ?.winName;
     const mainWindow = windowHandler.getMainWindow();
-    if (hideOnCapture) {
-      const curWindow = getWindowByName(currentWindow || '');
-      const mainWindow = windowHandler.getMainWindow();
-      mainWindow?.minimize();
-      if (currentWindow !== 'main') {
-        curWindow?.minimize();
+
+    if (windowObj.windows.length < 1) {
+      const allWindows = BrowserWindow.getAllWindows();
+      const windowsArr: IWindowState[] = [
+        {
+          id: 'main',
+          focused: mainWindow?.isFocused(),
+          minimized: mainWindow?.isMinimized(),
```

Comment:
We don't actually need it now. I forgot to remove it.
Since rightnow you will show the old windows without changing anything relating to the aspect of windows